### DISCUSSION TITLE
成功部署OctoMation的环境，请继续补充

### DISCUSSION THREAD

**Discussion by wzfukui** [flagify-com/OctoMation#20] [Show and tell]
经过人肉测试，以下环境安装部署OctoMation正常，供参考：

| 平台  | 操作系统            | CPU | 内存  | 磁盘   | Docker | Docker Compose | 其它 |
|-----|-----------------|-----|-----|------|--------|----------------|----|
| 华为云 | CentOS 8.5.2111 | 2C  | 8GB | 80GB | 24.0.6 | 1.29.2         | /  |
| 华为云 | CentOS 8.5.2111 | 2C  | 8GB | 80GB | 24.0.6 | 2.20.2         | /  |
| 阿里云 | CentOS 7.8.2003 | 2C  | 8GB | 80GB | 24.0.6 | 1.29.2         | /  |
| 腾讯云 | CentOS 7.9.2009| 2C  | 8GB | 80GB | 24.0.6 | 1.29.2         | /  |
| HP 笔记本电脑 | CentOS 7.9| 2C  | 8GB | 100GB | 24.0.6 | 1.29.2         | VMWare Workstation |
|DELL 工作站 | CentOS 7.9.2009| 2C  | 10GB | 150GB | 24.0.6 | 1.29.2         | VMWare ESXi |

**以上环境仅仅是为了验证OctoMation可以安装部署，实际在使用过程中务必提高资源配额以满足生产使用需求**

**Comment by williamdeve** [comment]
hp笔记本  centos7.9（win10中vmware workstation安装的虚拟机）2c 8g 100gb

**Reply by xuliang5114923** [reply]
dell 工作站， VMWare ESXi，CentOS 7.9.2009  2C 10G 150G

**Reply by wuzhi-dev** [reply]
thanks

**Comment by williamdeve** [comment]
qemu-kvm  Ubuntu 22.04  linux7.9.2009  2c 8g

**Comment by wuzhi-dev** [comment]
HP 笔记本电脑 | CentOS stream 8 | 2C | 8GB | 100GB | 24.0.6 | 1.29.2 | VMWare Workstation


在线安装时遇到未知操作系统错误，在ai_install.sh 313行增加：
elif [[ $(echo "$os_name" | grep -ic "CentOS Stream 8") == 1 ]]; then
    os_name="centosstream8"
重新到 /tmp/honeyguide_whole_package/ai_package_to_deploy_1.0.0/ai_package_to_deploy目录下执行bash ai_install.sh即可
